iron-form.html

```diff
@@ -288,7 +295,8 @@
         // were already added as a custom element, they don't need
         // to be re-added.
         if (!this._useValue(el) ||
-            (el.hasAttribute('is') && json[el.name])) {
+            (el.hasAttribute('is') && json[el.name]) || 
```

Comment:
Ideally yes, but that would change the current behavior which is at the moment - if two custom elements have the same name, the values are added to an array and passed to the server as comma separated values.